.circleci/config.yml

```diff
@@ -9,9 +9,34 @@ job_defaults: &job_defaults
   shell: /bin/bash --login
 
 test_containers:
+  - &job_parameters
+    parameters:
+      ruby_version:
+        description: Ruby version
+        type: string
+      image:
+        description: Docker image location
+        type: string
+      jit:
+        description: Jit enabled?
+        type: boolean
+        default: false
```

Comment:
Because I had to add a new variant `3.0 with jit`, I found it easier to extract the `parameters:` block into a shared snippet.

With this I noticed a few more copy/paste patterns in this configuration and extracted the ones that were trivial (`docker` and `environment`).